#landing-page: users landing page

nraufu opened the pull request:
> #### What does this PR do?
- It creates a landing page for users of the website
> #### Description of Task to be completed
- add folder structure (css,js,index.html)
- add a navbar with links to visit other pages
- add h1 tags containing small information about the developer
> #### How should this be manually tested?
- git clone the project
- open the index.html file in the browser you should see the page
> #### What are the relevant Trello stories?
- https://trello.com/c/EcWmS32m/1-user-landing-page
#### Screenshots (if appropriate)
![landing__page](https://user-images.githubusercontent.com/49249449/87689736-dcac9900-c788-11ea-8107-e8b88a8b9b80.PNG)

nraufu committed:
ft(landing-page): added landing page
- add folder structure (css,js,index.html)
- add a navbar with links to visit other pages
- add h1 tags containing small information about me

[finishes #landing page]

T2Wil commented:
Hey @nraufu , 
You did nice work, here are the few feedback you might consider working on:
- on branch name: consider adding story id.
- in PR description: add your git link to make it easy for a reviewer cloning your repo.
- Mobile responsiveness: make the web responsive on small devices.

T2Wil commented on UI/css/style.css:
```diff
@@ -0,0 +1,155 @@
+    justify-content: space-between;
+    background-color: rgba(255, 255, 255, 0.8);
+    box-shadow: 0 .2rem 2rem rgba(0, 0, 0, 0.25);
+    z-index: 500;
```
why is z-index having such big value?

nraufu replied:
I just putted it like to verflow those three bar image.

nraufu replied:
okay let start the mobile responsive part

nraufu commented:
![](https://github.trello.services/images/mini-trello-icon.png) [User Landing Page](https://trello.com/c/EcWmS32m/1-user-landing-page)

nraufu committed:
ft(landing-page): added responsiveness feature to the page
- add media query for different screen devices responsiveness

[finishes #EcWmS32m]
